packages/components/pages/f-registration/package.json

```diff
@@ -51,14 +51,14 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.4.0",
+    "@justeat/f-services": "1.x",
     "vuelidate": "0.7.6"
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "@justeat/f-button": "3.x",
+    "@justeat/f-button": "4.x",
     "@justeat/f-card": "3.x",
-    "@justeat/f-error-message": "1.x",
+    "@justeat/f-error-message": "2.x",
```

Comment:
Although this updates to a new major version, the only thing that's been added to `f-error-message` is new Sass syntax. This dependency will continue to function as it did before.